templates/airtable/.nango/schema.ts

```
@@ -0,0 +1,86 @@
+// ---------------------------
+
+export interface SyncMetadata_airtable_tables {
+};
+
+export interface Table {
+  baseId: string;
+  baseName: string;
+  id: string;
+  name: string;
+  views: (string | string | string)[];
+  fields: (string | string | string | string | {})[];
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)